### PR TITLE
Ported ScoringAction Low and Upper Hub

### PR DESCRIPTION
**Pull request by amull23**

### CODE REVIEW

File: Offseason2022/src/main/java/frc/robot/commands/ScoringActionHighHub.java [hunk 1]
```diff
@@ -3,27 +3,42 @@
...
-    // new command2(argsN, subsystem),
-    // new command3(argsN, subsystem)
-    // )
-    // );
+        // Add Commands here:
+
+        // @formatter:off
+      new ParallelDeadlineGroup(
+        new WaitUntilCommand(shooter::isAtDesiredSpeed), 
+        new ShooterRun(shooter, Mode.SHOOTER_UPPERHUB)),
```
Comment (jmullins16) [inline Offseason2022/src/main/java/frc/robot/commands/ScoringActionHighHub.java:34]: Let's use the style you have below. Swing the second parens on line 34 around and put it on its own line. Everything else looks good. You'll see the same issue in both LowHub and HighHub.

**Review by jmullins16 — CHANGES_REQUESTED** [review]
Just that minor change, then I'll approve and it's all good.

File: Offseason2022/src/main/java/frc/robot/commands/ScoringActionHighHub.java (hunk 1 above)
Comment (jmullins16) [reply Offseason2022/src/main/java/frc/robot/commands/ScoringActionHighHub.java:34]: Except I see both using HighHub while one should be using constants and name for LowHub.

**Comment by jmullins16** [comment]
Looks like you haven't merged from main yet, either.

**Review by zaraansari17 — APPROVED** [review]
Looks good

**Review by jmullins16 — APPROVED** [review]
Looks correct now!

**Pull request merged**